hack/eks

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: terraform enviroment variables need to be set
+INST_PROFILE_NAME=suwang-eks-worker-nodes-1-NodeInstanceProfile-8LL239W4W3GN
+ROLE_NAME=k8s
+
+AWS_CLI="aws"
+TERRAFORM="terraform"
+KUBECTL="kubectl"
+AWS_IAM="aws-iam-authenticator"
+RUN_PATH="$HOME/tmp/terraform-eks"
+CONFIG_PATH=$(dirname $(readlink -f "$0"))/terraform/eks
+CMD_ARRAY=($AWS_CLI $TERRAFORM $KUBECTL $AWS_IAM)
```

Comment:
It seems to me that the only reason you require people to install the AWS CLI is so that the credentials file is available: https://www.terraform.io/docs/providers/aws/index.html#shared-credentials-file. I believe the cli itself is never actually used. That's fine by me but maybe worth mentioning somewhere (and maybe there is no need to actually check that the cli is installed as part of this script).

Reply:
aws_cli is used to check user does have permission to access iam_role and eks.